test/jdk/java/lang/reflect/AccessFlag/BasicAccessFlagTest.java

```diff
@@ -25,18 +25,24 @@
  * @test
  * @bug 8266670 8281463 8293626
  * @summary Basic tests of AccessFlag
+ * @modules java.base/jdk.internal.misc
+ * @enablePreview
+ * @run main/othervm --enable-preview BasicAccessFlagTest
```

Comment:
`@enablePreview` should always add `--enable-preview` to test execution.  If you want to test with preview enabled and disabled, it can't use `@enablePreview`